### ISSUE TITLE
ToggleSwitch blue color is cyan

### ISSUE THREAD

**Issue by vitalijalbu** [themesberg/flowbite-react#1435]
- [x] I have searched the [Issues](https://github.com/themesberg/flowbite-react/issues) to see if this bug has already been reported
- [x] I have tested the latest version

## Steps to reproduce

![Screenshot 2024-06-22 alle 12 56 47](https://github.com/themesberg/flowbite-react/assets/40239902/f720d65c-078c-4ac0-b006-fbadf51872e0)

## Current behavior

Cyan color isntead of blue... also primary color is not working

## Expected behavior

Must be blue

**Comment by vitalijalbu** [comment]
guys, this is not only for the **toggle component**, but all components has this issue

**Comment by dhavalveera** [comment]
Hey @vitalijalbu , can you please share where else you're encountering this cyan color issue?

**Comment by vitalijalbu** [comment]
> Hey @vitalijalbu , can you please share where else you're encountering this cyan color issue?

tabs, loading, focus on textinputs, almost everything

**Comment by vitalijalbu** [comment]
> > > Hey @vitalijalbu , can you please share where else you're encountering this cyan color issue?
> > 
> > 
> > tabs, spinner, focus on textinputs, almost everything
> 
> @vitalijalbu , cyan is kind of default color.
> 
> in the Issue, you created as in the color of `blue`, cyan value was passed which is fixed, else everywhere cyan is used correctly.
> 
> @SutuSebastian -- if I am wrong, please correct me

yes, cyan is used even if the color is blue or something else, meaning cyan is always present and doesn't change with your custom color

**Comment by vitali-bc** [comment]
![Screenshot 2024-07-11 alle 09 35 50](https://github.com/themesberg/flowbite-react/assets/172402530/96084ab5-740b-4885-a9bb-99b190a33126)

SELECT also keeps cyan @SutuSebastian @dhavalveera

**Comment by SutuSebastian** [comment]
Fixed by #1498 in [flowbite-react@0.11.0](https://github.com/themesberg/flowbite-react/releases/tag/flowbite-react%400.11.0) release.
